sdk-python/src/omnilog/send_to_db.py

```diff
@@ -14,7 +14,12 @@ def send_to_db(url, log):
 
 
 def connect_to_db(url):
-    conn = psycopg2.connect(url)
+    try:
+        conn = psycopg2.connect(url)
+    except psycopg2.OperationalError:
+        print(f"Could not connect to database: {url}\n")
```

Comment:
What happens if its not an OperationalError?

Reply:
Documentation mentions this
> OperationalError is a class of error encountered while working with the psycopg library in Python. Operational errors are closely linked with the connect method within psycopg and typically occur when parameters passed to this method are incorrect or invalid. This may mean that a parameter, such as a database name, is spelled incorrectly, has changed, or that the database server itself is experiencing a problem.
Thus I believe that other errors should be handled by the usual flow of debugging